gtk4/Gir.toml

```diff
@@ -776,6 +776,17 @@ name = "Gtk.CssSection"
 status = "generate"
 trust_return_value_nullability = false
 
+[[object]]
+name = "Gtk.CustomSorter"
```

Comment:
Remove the other definition of Gtk.CustomSorter in the Gir.toml file, otherwise this one isn't even used :)

Reply:
This wasn't resolved